docs/en/docs/advanced/extending-openapi.md

```diff
@@ -248,3 +248,27 @@ Now, to be able to test that everything works, create a *path operation*:
 Now, you should be able to disconnect your WiFi, go to your docs at <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>, and reload the page.
 
 And even without Internet, you would be able to see the docs for your API and interact with it.
+
+
+### Add Tags
+
+You can group path by tags in both docs and redoc by using tags, tags also allows you to put descriptions and links. Tags description suppports markdown.
+
+```Python hl_lines="9-22 56"
+{!../../../docs_src/extending_openapi/tutorial003.py!}
+```
+
+### Change the *path operation* order
+
+It is good to have a documentation, but a sorted one is better, you can use tags_sorter and operations_sorter to sort your docs.
+See this thread for the underlying logic (Swagger UI 3.17.6 required, Redoc not affected) :
+<a href="https://github.com/swagger-api/swagger-ui/issues/2990" class="external-link" target="_blank">https://github.com/swagger-api/swagger-ui/issues/2990</a>
+
+
+```Python hl_lines="37-38"
+{!../../../docs_src/extending_openapi/tutorial003.py!}
+```
+
+!!! tip
+    Possible values are "aplha" or a custom build *sort* function for the tags,
```

Comment:
A example method for "sort" would be 👍

Reply:
Doesn't need to be *that* visual, just changing around 1 endpoint would be enough.

Reply:
I wrote a function in a generic manner using the length as ordering as it is simple to understand, effectively changes the display order in a easy to check way and as a bonus it allows to show the way of getting each entry "name" which is the operation path.

```
     operations_sorter='function(e,t) {\
            if (e.get("path").length < t.get("path").length) {return -1;}\
            if (e.get("path").length > t.get("path").length) {return 1;}\
            return 0;\
            }',
```